src/shader_recompiler/backend/spirv/emit_context.cpp

```diff
@@ -430,10 +430,11 @@ Id DescType(EmitContext& ctx, Id sampled_type, Id pointer_type, u32 count) {
     }
 }
 
-size_t FindNextUnusedLocation(const std::bitset<IR::NUM_GENERICS>& used_locations,
-                              size_t start_offset) {
+size_t FindNextUnusedLocation(std::bitset<IR::NUM_GENERICS>& used_locations, size_t& start_offset) {
```

Comment:
```suggestion
size_t FindAndSetNextUnusedLocation(std::bitset<IR::NUM_GENERICS>& used_locations, size_t& start_offset) {
```

Since this is doing more than just finding now